contributed_definitions/NXspindispersion.nxdl.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<definition category="base" extends="NXobject" name="NXspindispersion" type="group" xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.w3.org/2001/XMLSchema-instance">
+    <doc>
+         Subclass of NXelectronanalyser to describe the spin filters in photoemission
+         experiments.
+    </doc>
+    <field name="spin_filter_type" type="NX_CHAR">
```

Comment:
`spin_` is a common prefix for all these fields.  Consider if you might drop it (for shorter names).

Reply:
As written, it is easier to locate in a search.